numbers/src/main/resources/config/fr-fr/tokenizer.json

```diff
@@ -0,0 +1,393 @@
+{
+  "spaces": " \t\n\f\r:;_!?<>|=()[]{}»«*~^`'\"",
+  "characters_as_word": "%‰#-+.,/",
+  "raw_number_categories": [
+    "number",
+    "raw"
+  ],
+  "plural_endings": [
+    "s"
+  ],
+  "word_matches": [
+    {
+      "categories": [
+        "ignore"
+      ],
+      "values": [
+        "et"
+      ]
+    },
+    {
+      "categories": [
+        "ignore",
+        "thousand_separator"
+      ],
+      "values": [
+        " ",
```

Comment:
A space can't be a word, since spaces are word separators. If you want to say that thousands can be separated by spaces (i.e. nothing), you need to do so in Java code.